src/runtime_src/core/pcie/noop/pcidev_noop.cxx

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright (C) 2023 Advanced Micro Devices, Inc. All rights reserved.
+
+#include "pcidev_noop.h"
+
+namespace xrt_core { namespace pci {
```

Comment:
The namespace should be 
```suggestion
namespace xrt_core::noop {
```
Your are in the `noop` module, so keep types for `noop` in that namespace.